app/views/idv/doc_auth/no_camera.html.erb

```diff
@@ -16,5 +16,5 @@
     <%= t('doc_auth.errors.no_camera.explanation') %>
   </p>
 
-  <%= render 'idv/doc_auth/start_over_or_cancel' %>
+  <%= render 'idv/doc_auth/start_over_or_cancel', step: 'device_choice_no_camera' %>
```

Comment:
This is a bit of an outlier in that it's not really a step, but instead a warning screen where the user could drop out. It might make a bit more sense to just tie this into the `device_choice` step, though I wasn't sure if it could help to be able to differentiate between users choosing to cancel from the choice vs. canceling from the warning that they don't have a supported camera.